### ISSUE TITLE
Do you rewrite some baselines (such as CFR, SITE, DR-CFR, CEVAE) with PyTorch?

### ISSUE THREAD

**Issue by xyzanonymous666** [WeijiaZhang24/TEDVAE#4]
Dear Zhang,
Thanks for releasing your code. We can see you reported the results of some baselines (such as CFR, SITE, DR-CFR, CEVAE) in IHDP datasets which are generated by yourselves. However, these results are not consistent with that in the original references. Thus, I have a question if you reproduce these methods with PyTorch. If so, could you provide your rewritten code? Thanks so much!

**Comment by xyzanonymous666** [comment]
Plus, CFR has three versions, one without imbalance loss (i.e., TARNet), two with imbalance loss (i.e., CFR MMD and CFR Wass). Which version did you report in your paper?

**Comment by WeijiaZhang24** [comment]
CEVAE has PyTorch implementation using Pyro (the same code structure with TEDVAE). CFR and its variants all used older versions of Tensorflow. If memory serves, they use TF v1.07 or so. 

I had a look on the results of various CFR variants reported in DR-CFR, they are consistent to the results reported in TEDVAE. If by inconsistency you mean the results of CEVAE compared in the original paper, the Pyro implementation does seem to perform better than the original CEVAE implementation provided by Louizos.
